src/main/java/Nakhanov/REST/REST_3rd_Day/PetController/PetController.java

```diff
@@ -0,0 +1,105 @@
+package Nakhanov.REST.REST_3rd_Day.PetController;
+
+
+import Nakhanov.REST.PetDTO.Pet;
+import com.jayway.restassured.response.ValidatableResponse;
+
+import java.util.Map;
+
+import static com.jayway.restassured.RestAssured.baseURI;
+import static com.jayway.restassured.RestAssured.given;
+
+public class PetController {
+
+
+    public static void addNewPetEndpoint(Pet pet){
+        given().spec(Utils.setSpec())
+                .body(pet)
+                .when()
+                .post(baseURI)
+                .then()
+                .statusCode(200);
+    }
+
+    public static Pet getPetByIdEndpoint(long id){
+        Pet pet = given().spec(Utils.setSpec())
+                .when()
+                .get(baseURI + id)
+                .then()
+                .statusCode(200)
+                .extract()
+                .as(Pet.class);
+        return pet;
+    }
+
+    public static Pet putPetEndpoint(Pet newPet){
+        Pet pet = given().spec(Utils.setSpec())
+                .body(newPet)
+                .when()
+                .put(baseURI)
+                .then()
+                .statusCode(200)
+                .extract()
+                .as(Pet.class);
+        return pet;
+    }
+
+    public static Pet[] getPetByStatusEndpoint(String status){
+        Pet[] pet = given().spec(Utils.setSpec())
+                .queryParam("status", status)
+                .when()
+                .get(baseURI + "findByStatus")
+                .then()
+                .statusCode(200)
+                .extract()
+                .as(Pet[].class);
+//        List<Pet> petList = Arrays.stream(pet).collect(Collectors.toList());
+//        List<Pet> petList = Arrays.asList(pet);
+        return pet;
+    }
+
+    public static Pet[] getPetByStatusEndpoint2(Map<String, String> map){
+        return temp("pet/findByStatus", map).extract().as(Pet[].class);
+    }
+
+    public static Pet[] getPetByStatusEndpoint3(Map<String, String> map){
```

Comment:
It's better to name "getPetByStatusEndpoint" with different ways, not adding 1, 2, 3...

Reply:
this is Andrey code, just for testing